src/templates/events/EventDirectionsSection.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import querystring from 'querystring'
+
+import { Container, Row } from '../../components/grid/grid'
+import { media } from '../../theme/media'
+import theme from '../../theme/theme'
+
+const StyledContainer = styled(Container)`
+  padding: 0px 0px 30px;
+  background-color: white;
+  ${media.desktop`
+    padding: 60px 0px;
+    background-color: ${theme.colors.indigo};
+  `};
+`
+
+const Heading = styled.h2`
+  font-size: 1.125rem;
+  font-weight: 300;
+  margin: 0px 10px 15px 10px;
+  color: ${props => props.theme.colors.indigo};
+  ${media.desktop`
+    margin: 0px 15px 15px 15px;
+    font-size: 1.5rem;
+    font-weight: 600;
+    margin-bottom: 30px;
+    color: white;
+  `};
+`
+
+const MapLink = styled.a`
+  display: block;
+  height: 215px;
+  background-size: cover;
+  background-position: center center;
+  margin: 0px;
+  ${media.tablet`
+    height: 350px;
+    margin: 0px 50px;
+  `};
+  ${media.desktop`
+    height: 450px;
+    max-width: 830px;
+    margin: 0px 90px;
+  `};
+`
+
+export class EventDirectionSection extends React.Component {
+  state = { width: null, height: null }
+
+  componentDidMount() {
+    this.componentDidUpdate()
+  }
+
+  componentDidUpdate() {
+    const rect = this.wrapperRef.getBoundingClientRect()
```

Comment:
The map 'link' is sized with css, and then we measure it after it was mounted so we can request an image of a reasonable size from Google.